docs/snippets/package/alibaba/crossplane.yaml

```diff
@@ -5,3 +5,9 @@ metadata:
   annotations:
     guide: quickstart
     provider: alibaba
+spec:
+  crossplane:
+    version: ">=v0.14.0-0"
```

Comment:
is the expectation that we'll update these in the release branch for each release?

Reply:
@jbw976 no I don't think so, only when these packages themselves get updated. This constraint will use the latest version that is above v0.14.0, so we will roll forward to any new releases.

Reply:
It might make sense to pin them to v1.0.0 when it is available, but they would functionally operate the same

Reply:
does using the trailing `-0` pre-release identifier in the constraint imply that we'd pick up the latest pre-release? if so, that would make sense in master, but perhaps not from the release branch docs.

Reply:
hmm I think it would be good on release branches as well. The release branches have the documentation for both installing Crossplane from stable or from master. It is likely that folks will be installing stable, but if they do want to try from master I would prefer that work as well. What do you think?

Reply:
i think that makes sense, i just wasn't sure how the dependency resolution aspect of this works...so say someone is following the release branch documentation and they haven't yet installed provider-alibaba yet when they install this snippet.  would dependency resolution give them the latest bleeding edge build from master, or would it give them the latest official release or provider-alibaba?

Reply:
@jbw976 so that is switching gears a bit because the constraint highlighted here is for `crossplane` version, rather than the dependencies. I think `crossplane` constraint should keep the prerelease `-0`. For the dependencies, there is a trade-off:
- Typically folks will have already installed the providers when going through the docs sequentially (at least for now, maybe we will update to just having them auto-installed by dep resolution in the future). This means that they will have chosen their version, typically the latest stable. The version constraints allowing a pre-release for the dependency will work with this installed stable release, but also allow for other non-stable versions if someone wanted to test them out.
- If dependency resolution is used, we will pull the in the latest from master for the provider, which could not be the experience folks want.

For now, while we are not using dependency resolution, it seems like it may be optimal to allow pre-release versions. We could potentially update versions on the release branch as you mentioned to just not include `-0`.

Reply:
Got it, OK that thinking makes sense to me.  Thank you for explaining Dan!